read.go

```diff
@@ -256,9 +256,9 @@ func (r *reader) readPixelData(vl uint32, d *Dataset, fc chan<- *frame.Frame) (V
 
 			if fc != nil {
 				fc <- &f
+			} else {
+				image.Frames = append(image.Frames, &f)
 			}
```

Comment:
I suppose this is also an API change that means the PixelDataInfo returned from Parse will not have frames populated if the framechannel option is used? This might be ok, but is a meaningful change I'll think about a little.